### PR TITLE
Use black formatter for python code

### PR DESCRIPTION
**Pull request by cbachhuber**
[black](https://github.com/psf/black) is a widely used python formatter, let's go for that in CI.

### CODE REVIEW

File: delayrecorder.py [hunk 1]
```diff
@@ -71,44 +101,61 @@ def get_measurements_serial(num_measurements, quiet_mode):
...
+                    "Did not receive msmt data from the Arduino for another 5 seconds. "
+                    "Is the phototransistor still sensing the LED?"
+                )
             else:
-                print("Did not receive msmt data from the Arduino for 5 seconds. \n Is the phototransistor sensing the LED on the screen? \n Is the correct side of the PT pointing towards the screen (the flat side with the knob on it)? \n Is the screen brightness high enough (max recommended)?")
+                print(
+                    """Did not receive msmt data from the Arduino for 5 seconds.
+Is the phototransistor sensing the LED on the screen?
+Is the correct side of the PT pointing towards the screen (the flat side with the knob on it)?
+Is the screen brightness high enough (max recommended)?"""
```
[the remark this answers is not in the document]
Comment (cbachhuber) [reply delayrecorder.py:128]: Great, thanks for checking!

[the remark this answers is not in the document]
Comment (cbachhuber) [reply delayrecorder.py:128]: I'll check extracting the string in a follow-up PR, when I'll anyways clean the python code some more.

**Comment by vashmata** [comment]
I can't make the change myself but you could add `--color` to the command and it will highlight the diff if the code needs to be formatted

**Comment by cbachhuber** [comment]
> I can't make the change myself but you could add --color to the command and it will highlight the diff in colour (red and green in my case) if the code needs to be formatted

Cool idea, implemented in #21. Please review :)

**Pull request merged**